sdk/Sdk.Generators/FunctionMetadataProviderGenerator/FunctionMetadataProviderGenerator.Parser.cs

```diff
@@ -573,7 +573,7 @@ private bool TryCreateBindingDict(AttributeData bindingAttrData, string bindingN
                     }
                     else
                     {
-                        bindings[propertyName] = prop.Value!.ToString();
+                        bindings[propertyName] = prop.Value!; // this value has already been parsed by "TryParseValueByType" during the call to "TryGetAttributeProperties" earlier in this method
```

Comment:
nit: not sure this comment here is helpful. I think it's clear what the behavior here is once this is serialized and this comment would become confusing if things are ever refactored.